app/client/components/payment/directDebitDisplay.tsx

```diff
@@ -119,32 +108,20 @@ export const DirectDebitDisplay = (mandate: DirectDebitDisplayProps) => (
   <div
     css={{
       display: mandate.inline ? "inline-flex" : "flex",
+      flexWrap: "wrap",
       alignItems: "top"
     }}
   >
-    <DirectDebitLogo fill={palette.neutral[7]} />
+    <DirectDebitLogo fill={palette.neutral[7]} marginRight />
     <div
       css={{
         display: "flex",
         flexWrap: "wrap"
       }}
     >
-      <span
-        css={{
-          marginLeft: "10px"
-        }}
-      >
-        {dashifySortCode(mandate.sortCode)}
-      </span>
```

Comment:
I think we need to speak to Jian about losing the display of the sort code (as this is helpful in identifying the account being used)?

Reply:
I see you've added back in (https://github.com/guardian/manage-frontend/pull/413/commits/db749c624f82970794b954d0e28ea3342028b591) - probably want to update description and screenshots

Reply:
Jian agreed to add the sortcode back in. will update the description accordingly